dockerfiles/QAT/qat.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_PCIEAER=y
+CONFIG_UIO=y
+CONFIG_CRYPTO_HW=y
+CONFIG_CRYPTO_DEV_QAT_C62XVF=m
```

Comment:
Do we mean a module?

Reply:
@mythi @eadamsintel - do you have a definitive answer? I suspect the answer is 'yes' for some reason or other (I wanted to say 'out of tree driver', but this is of course the kernel config...)

Reply:
yes, a module. it's a trick: by setting `=m` Kconfig enables the rest of the QAT dependencies we need. it cannot be `y=` because we don't want that driver but the out-of-tree one we build to rootfs later.

Reply:
heh, I'll see if I can add a comment to that effect - I think I can put # comments in the fragments..... as long as they don't explicitly mention `CONFIG_...` names (as the magic tool greps and breaks at that point iirc).